lib/screens/onboarding/create_account.dart

```diff
@@ -76,53 +77,98 @@ class _CreateAccountState extends State<CreateAccount> {
   }
 
   String _validateAccountName(String val) {
-    if (val.length != 12)
+    if (val.length != 12) {
       return 'Your account name should have exactly 12 symbols';
-    if (RegExp(r'0|6|7|8|9').allMatches(val).length > 0)
+    } else if (RegExp(r'0|6|7|8|9').allMatches(val).length > 0) {
```

Comment:
also need to check that account name starts from letter (not from digit)

Reply:
good catch.